Example/Example-Info.plist.license

```diff
@@ -0,0 +1,5 @@
+This source file is part of the ResearchKitOnFHIR open source project
```

Comment:
Remove this file once the `.plist` file is removed.

Reply:
Removed in [c7d76ef](https://github.com/StanfordBDHG/ResearchKitOnFHIR/pull/59/commits/c7d76efcf6d5367a838f0df9a6559f719d71bf9a)